bin/generate-podspecs.sh

```diff
@@ -161,23 +163,30 @@ do
         fi
 
         # Generate React-Codegen
-        # A copy of react_native_pods is done to modify the content within get_react_codegen_spec
+        # A copy of codegen_utils.rb is done to modify the content within get_react_codegen_spec
         # this enables getting the schema for React-Codegen in runtime by printing the content.
         echo "Generating React-Codegen"
-        REACT_NATIVE_PODS_PATH="$SCRIPTS_PATH/react_native_pods.rb"
-        REACT_NATIVE_PODS_MODIFIED_PATH="$SCRIPTS_PATH/react_native_pods_modified.rb"
-        # Making a temp copy of react_native_pods.rb
-        cp $REACT_NATIVE_PODS_PATH $REACT_NATIVE_PODS_MODIFIED_PATH
+        REACT_NATIVE_CODEGEN_UTILS_PATH="$SCRIPTS_PATH/cocoapods/codegen_utils.rb"
+        REACT_NATIVE_CODEGEN_UTILS_MODIFIED_PATH="$SCRIPTS_PATH/cocoapods/codegen_utils_modified.rb"
+        # Making a temp copy of codegen_utils.rb
+        cp $REACT_NATIVE_CODEGEN_UTILS_PATH $REACT_NATIVE_CODEGEN_UTILS_MODIFIED_PATH
+        # Manually add the min_ios_version_supported variable to the CodegenUtils class
+        # The modified script won't be able to detect min_ios_version_supported otherwise
+        echo "
+        def min_ios_version_supported
+        '12.4'
```

Comment:
Eventually, we should fetch this value from React Native configuration instead of setting it fixed ([reference](https://github.com/facebook/react-native/blob/7ea7d946c643f076c29bcf11b927f7569e3c516f/scripts/react_native_pods.rb#L26-L31)).